sound/soc/sof/core.c

```diff
@@ -512,9 +517,14 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 	INIT_LIST_HEAD(&sdev->widget_list);
 	INIT_LIST_HEAD(&sdev->dai_list);
 	INIT_LIST_HEAD(&sdev->route_list);
+	INIT_LIST_HEAD(&sdev->vbe_list);
```

Comment:
Why would we have more than 1 BE ? I'd expect we would have a single BE and multiple FEs.

Reply:
@lgirdwood This is a list of BE instances, allocated when the misc device is opened, which I expect to happen every time a new QEMU instance is started? As long as we only support one guest, this list isn't needed of course, and at the moment we don't support more than one guest. So, for now we can remove this. But if we ever decide to support multiple guests, each of them will have its own set of virtual queues, right?

Reply:
I would expect each Qemu virtualised driver to be the vFE and the host driver to be the vBE so I dont quite understand why we need > 1 vBE ?

Reply:
@lgirdwood I'm sure I've explained this on a couple of occasions too, at least in The Doc - these are multiple VMs open()ing the same BE misc device node